.github/workflows/build.yml

```diff
@@ -0,0 +1,96 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Unit Tests
+
+    strategy:
+      matrix:
+        node-version:
+          - 12.x
+          - 14.x
+        os:
+          - ubuntu-latest
+          - windows-latest
+          - macOS-latest
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+    
+      - run: npm ci
+      - run: npm run build --if-present
+      - run: npm test
+
+      - name: Calculate code coverage
+        run: npm run coverage
+
+      - name: Coveralls
+        uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          flag-name: ${{ matrix.os }}
+          parallel: true
+
+  notify:
+    needs:
+      - build
+
+    name: Code Coverage
+    if: ${{ success() }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Coveralls Finished
+        uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          parallel-finished: true
+
+  publish:
+    needs:
+      - build
+      - notify
+
+    name: Publish to npm
+    if: ${{ success() && github.event_name == 'push' && github.repository_owner == 'accordproject' }}
+    runs-on: ubuntu-latest
+
+    outputs:
+      job-status: ${{ job.status }}
+
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v2
+
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+    
+      - name: Build
+        run: npx lerna bootstrap
```

Comment:
Should I replace `npx lerna bootstrap` with `npm run build` ?

Reply:
this looks like the wrong script to me. The latest version should be at: https://github.com/accordproject/concerto/tree/master/.github/workflows

Reply:
Can you look the file again it is same as https://github.com/accordproject/concerto/blob/master/.github/workflows/build.yml